c-sharp-server/c-sharp-client/Program.cs

```diff
@@ -6,48 +6,67 @@ class AuthDelegate : IDittoAuthenticationDelegate
 {
     public async void AuthenticationRequired(DittoAuthenticator authenticator)
     {
-        System.Console.WriteLine($"Login request");
+        Console.ForegroundColor = ConsoleColor.Red;
+        Console.WriteLine($"\nAuth required!! → try \"--login jellybeans\"");
+        Console.ResetColor();
     }
 
     public async void AuthenticationExpiringSoon(DittoAuthenticator authenticator, long secondsRemaining)
     {
-        System.Console.WriteLine($"Auth token expiring in {secondsRemaining} seconds");
+        Console.WriteLine($"Auth token expiring in {secondsRemaining} seconds");
     }
 }
 
-namespace Program {
+namespace Program
+{
 
     class App {
         static Ditto ditto;
-        static DittoSubscription subscription;
-        static DittoLiveQuery liveQuery;
+
+        static IDisposable dittoAuthObserver;
```

Comment:
Probably no need to keep the static reference.

Reply:
The original app used `static` for properties. It looks like `Main` has to be `static`, thus all the properties have to be `static`.

`CSC : error CS5001: Program does not contain a static 'Main' method suitable for an entry point [/Users/shunsukekondo/repos/ditto/samples/c-sharp-server/c-sharp-client/Program.csproj]`

<img width="670" alt="image" src="https://github.com/getditto/samples/assets/26117257/be2ea08c-897d-4c3c-8bb5-f17ecda2ff30">